app/controllers/donates_controller.rb

```diff
@@ -0,0 +1,29 @@
+class DonatesController < ApplicationController
```

Comment:
I suggest to use `DonationsController` as name, because our domain model is `Donation`